.github/workflows/deploy.yml

```diff
@@ -52,28 +51,20 @@ jobs:
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           environment: dev
           preCommands: npx wrangler deploy --env dev # https://github.com/cloudflare/wrangler-action/issues/162
+          postCommands: npx wrangler delete --name wrangler-action-dev-environment-test --force
           secrets: |
             SECRET1
             SECRET2
         env:
           SECRET1: ${{ secrets.SECRET1 }}
           SECRET2: ${{ secrets.SECRET2 }}
 
-      - name: Clean up Deployed Environment Worker
-        uses: ./
-        with:
-          workingDirectory: "./test/base"
-          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
-          command: delete --name wrangler-action-dev-environment-test --force
```

Comment:
Is this actually a test we want to do? i.e. run the delete command?

Reply:
That's a great question. We actually do have a `delete` command below when cleaing up the secret test workers. I figured this change along with another one down below would give us the chance to test out `postCommands`. Since there wasn't a previous use of it in this script.